clang/test/dpct/cooperative_groups_reduce.cu

```diff
@@ -12,6 +12,8 @@ namespace cg = cooperative_groups;
 __device__ void testReduce(double *sdata, const cg::thread_block &cta) {
   const unsigned int tid = cta.thread_rank();
   cg::thread_block_tile<32> tile32 = cg::tiled_partition<32>(cta);
+  // CHECK: DPCT1119:{{[0-9]+}}: cooperative_groups::__v1::thread_block_tile<16> has not supported yet, please try to migrate with the DPCT experimental option: --use-experimental-features=logical-group.
+  cg::thread_block_tile<16> tile16 = cg::tiled_partition<16>(cta);
```

Comment:
reqd_sub_group_size(16) should work, while it may conflict with other API which require to set 32.

Reply:
Currently, the tool migrates the partition size 32's case to the reqd_sub_group_size(32). Other size group is simulated by the logical group defined in the helper function.

Reply:
I prefer to keep the current implementation to align with all the reqd_sub_group_size(32), and explicitly specified group size in the helper function interface.